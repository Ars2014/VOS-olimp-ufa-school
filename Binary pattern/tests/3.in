42
53
55
56
57
