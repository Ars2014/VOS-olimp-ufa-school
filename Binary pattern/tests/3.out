Pattern
Pattern
No
No
No
